### PR TITLE
document how to install a specific version

### LINKED ISSUE #782: Missing Documentation for Installing Previous versions
# Summary
Documentation is missing for installing previous versions of REX-Ray. In addition, documentation for [v0.3.3](https://rexray.readthedocs.io/en/stable/user-guide/installation/#rex-ray-033) installation is no longer correct. It shows the current stable docs.

## Expected Behavior
I should be able to go to the documentation and see the syntax on how to install an older version

```
curl -sSL https://dl.bintray.com/emccode/rexray/install | sh -s -- stable 0.7.0
```

This is documented in the Install script, but not in the docs.

### PR DESCRIPTION
**Pull request by cantbewong**
Closes #782

### CODE REVIEW

**Comment by codenrhoden** [comment]
👍  This looks good to me. Rebase against the current master when you have the chance.

File: .docs/user-guide/installation.md [hunk 1]
```diff
@@ -40,6 +40,20 @@ specify `stable`, see the additional version names below that are also valid.
 curl -sSL https://dl.bintray.com/emccode/rexray/install | sh -s -- stable
 ```
 
+#### Install a specific version
```
Comment (akutz) [inline .docs/user-guide/installation.md]: Hi @cantbewong,

Please make this an `h3` instead of `h4`.

File: .docs/user-guide/installation.md [hunk 2]
```diff
@@ -40,6 +40,20 @@ specify `stable`, see the additional version names below that are also valid.
...
+
```
Comment (akutz) [inline .docs/user-guide/installation.md]: Hi @cantbewong,

Please be consistent with the rest of the documentation and omit a blank line between a heading and the paragraph that follows it.

File: .docs/user-guide/installation.md [hunk 3]
```diff
@@ -40,6 +40,20 @@ specify `stable`, see the additional version names below that are also valid.
...
+You can forcibly install a particular version like this:
```
Comment (akutz) [inline .docs/user-guide/installation.md]: Hi @cantbewong,

Please avoid the use of words like _you_, _I_, _we_, etc. In general the documentation should not be conversational and avoid the use of second person. This [guide](http://www.technicalauthoring.com/wiki/index.php/Using_the_correct_language#Descriptive_writing_should_be_in_the_third-person.2C_active_voice.2C_indicative) may prove useful.

An alternative approach is:

> A specific version of REX-Ray may be installed with the following command:

Please note that the word  _forcibly_ was intentionally removed. It's misleading as the installation follows the same rules as any other type, and thus is not more forceful than a standard install.

File: .docs/user-guide/installation.md [hunk 4]
```diff
@@ -40,6 +40,20 @@ specify `stable`, see the additional version names below that are also valid.
...
+
+```sh
+$ curl -sSL https://dl.bintray.com/emccode/rexray/install | sh -s -- stable 0.7.0
+```
+
+The version shows during an install, but to see it again later:
```
Comment (akutz) [inline .docs/user-guide/installation.md]: Hi @cantbewong,

Perhaps a cleaner approach to the above statement might be:

> REX-Ray's version is displayed during installation and can also be recalled at any time with the following command:

File: .docs/user-guide/installation.md (hunk 1 above)
Comment (cantbewong) [inline .docs/user-guide/installation.md]: now h3

File: .docs/user-guide/installation.md (hunk 2 above)
Comment (cantbewong) [inline .docs/user-guide/installation.md]: blank line removed

File: .docs/user-guide/installation.md (hunk 3 above)
Comment (cantbewong) [inline .docs/user-guide/installation.md]: I do not agree with the principle of avoiding the word "you". 

For example,  The use of "You" not only complies, but is recommend in the Kubernetes style [guide](https://kubernetes.io/docs/home/contribute/style-guide/#address-the-reader-as-you) for Kubernetes, 

The Mesos project style guide is simply a statement that the project uses the Google style guide. This seems to be a recognition of the Google guide as a quality authority on documentation.

From a perspective going beyond the container space, as part of the Material Design effort, Google maintains a huge, respected and well funded staff to research and test design principles. The [guide](https://material.io/guidelines/style/writing.html#writing-language) for authors of apps running on google platforms may prove to be useful. It specifically recommends the word "you".

The Docker Inc documentation style guide incorporates the AP style guide by reference. The AP guide is paywalled, but in reference to the AP guide, Docker's [style guide](https://docs.docker.com/opensource/doc-style/) says this: "If **you** don't have a copy of (or online subscription to) the AP guide, **you** can almost always find an answer to a specific question by searching the web." My conclusion, Docker probably likes "you", although I confess I didn't spend money to confirm this 100% 

The CloudFoundry style [guide](http://docs.pivotal.io/cf_style.html#audience) explicitly says: you should use "you."

I do not agree with this request, and invite you to reconsider your position.
The guide appears to be contradict mainstream documentation style guides within the container scheduler space.
The word forcibly has been removed.

File: .docs/user-guide/installation.md (hunk 4 above)
Comment (cantbewong) [inline .docs/user-guide/installation.md]: I do not agree that this suggested rewording is "cleaner". And I did an A-B compare using  online readability analyzers. My version returned significantly better scores.

I used this: https://readable.io/text/
And this: https://www.webpagefx.com/tools/read-able/check.php

**Comment by akutz** [comment]
Hi @koensayr,

This was merged into the wrong branch, and my review was outstanding and thus this should not have been merged anyway. I have to revert it and/or merge it up to the release branch. In the future, please check before merging a PR during a release window. Thanks!

**Comment by codenrhoden** [comment]
Shoot, this was my fault. Apologies to both @koensayr and @akutz. @koensayr asked me if there were any old open PRs he could take a look at, and I directed him to this one. It was open for so long, I completely forgot about the branch change.

**Pull request merged**